monkey/infection_monkey/dropper.py

```diff
@@ -134,7 +134,9 @@ def start(self):
                                                       'monkey_commandline': inner_monkey_cmdline}
 
         monkey_process = subprocess.Popen(monkey_cmdline, shell=True,
-                                          stdin=None, stdout=None, stderr=None,
+                                          stdin=subprocess.DEVNULL,
+                                          stdout=subprocess.DEVNULL,
+                                          stderr=subprocess.DEVNULL,
```

Comment:
Not sure this is related, but make sure this doesn't affect monkey and dropper logs.

Reply:
It causes the monkey's (not dropper's) log messages sent to stdout to go to `/dev/null` instead. This only the case when the dropper is used, and the log messages sent to a file are unaffected.

I switched to `subprocess.PIPE` instead. It seems to resolve the issue but still allows the monkey to log messages to stdout.